app/services/instrumentation.rb

```diff
@@ -0,0 +1,37 @@
+module Instrumentation
+  class << self
+    # The payload is optional so that we don't need an additional method
+    # just to read back what we've got stored.
+    def append_to_log(optional_payload = {})
+      RequestStore.store[:custom_log_items] ||= {}
+      RequestStore.store[:custom_log_items].
+        merge!(optional_payload) unless optional_payload.blank?
+      RequestStore.store[:custom_log_items]
+    end
+    alias_method :custom_log_items, :append_to_log
```

Comment:
@tatyree , following on from https://github.com/ministryofjustice/prison-visits-public/pull/29#discussion_r60395711 : my point was that the arguments to append_to_log need not be optional if custom_log_items is not an alias (so renaming args does not address my concern). Why not this

```
def append_to_log(payload)
  RequestStore.store[:custom_log_items] ||= {}
  RequestStore.store[:custom_log_items].merge!(payload)
end

def custom_log_items
  RequestStore.store[:custom_log_items]
end
```